views/partials/cards/article-card.twig

```diff
@@ -13,7 +13,7 @@
         {% endif %}
 		<div class="card-article__image">
 			{% set image = get_image(post.thumbnail()) %}
```

Comment:
Dont need get_image here just pass post.thumbnail to the image partial

Reply:
updated